scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -425,23 +485,34 @@ Function RunQuickstartCertsTest
         -n `"$env:computername`" ``
         -r `"$ContainerRegistry`" ``
         -u `"$ContainerRegistryUsername`" ``
-        -p `"$ContainerRegistryPassword`" --optimize_for_performance true ``
+        -p `"$ContainerRegistryPassword`" ``
         -t `"${ArtifactImageBuildNumber}-windows-$(GetImageArchitectureLabel)`" ``
-        --leave-running=Core ``
+        --optimize_for_performance true ``
+        --leave-running=All ``
         --no-verify"
+    If ($ProxyUri) {
+        $testCommand = "$testCommand ``
+            -l `"$DeploymentWorkingFilePath`" ``
+            --upstream-protocol 'AmqpWs' ``
```

Comment:
just curious, why need to use AmqpWs as upstream protocol for proxy scenario? does it support other protocol?

Reply:
It's an HTTPS proxy so it only supports HTTP(S) and WebSocket. We don't have proxy support for AMQP(S) or MQTT(S).

Reply:
thanks.